tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/stepdefinitions/throttling/ThrottlingStepDefs.java

```diff
@@ -0,0 +1,44 @@
+package io.hawt.tests.features.stepdefinitions.throttling;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.hawt.tests.features.pageobjects.pages.LoginPage;
+import io.hawt.tests.features.setup.LoginLogout;
+import org.openqa.selenium.By;
+
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Selenide.$;
+
+public class ThrottlingStepDefs {
+    private final LoginPage loginPage = new LoginPage();
+    private static final By WARNING = By.cssSelector("p.pf-c-form__helper-text.pf-m-error");
+
+    @Given("User is on Login page")
+    public void userIsOnLoginPage() {
+        LoginLogout.logout();
+        loginPage.loginPageIsOpened();
+    }
+
+    @When("the user attempts to log in with incorrect credentials {int} times")
+    public void theUserAttemptsToLogInWithIncorrectCredentialsTimes(int attempts) {
+        for (int i = 0; i < attempts; i++) {
+            loginPage.throttling("username", "invalid");
+        }
+
+    }
```

Comment:
The same step is on line 37, the only difference is just `credentials` -> `credential`, one can be removed